src/Model/Post/Media.php

```diff
@@ -1178,6 +1178,12 @@ public static function getUrlForId(int $id, string $size = ''): string
 	 */
 	public static function getAllocatedHeightByMedia(array $media): string
 	{
-		return (100 * $media['height'] / $media['width']) . '%';
+		if (!empty($media['height']) && !empty($media['width'])) {
+			return (100 * $media['height'] / $media['width']) . '%';
+		}
+		if (!empty($media['preview-height']) && !empty($media['preview-width'])) {
+			return (100 * $media['preview-height'] / $media['preview-width']) . '%';
+		}
+		return "100%";
```

Comment:
This means the image is a square, which isn’t necessarily the case. How come some widths are `0`?

Reply:
This ix just to prevent a division by zero.